lib/java/com/google/android/material/bottomsheet/res/values/styles.xml

```diff
@@ -44,6 +44,7 @@
     <item name="enforceMaterialTheme">true</item>
     <item name="android:background">@null</item>
     <item name="shapeAppearance">?attr/shapeAppearanceLargeComponent</item>
```

Comment:
We should be able to just set `<item name="shapeAppearance">@null</item>` to remove any rounded corners. Does that work?

Reply:
Well, shouldn't bottomsheet have large corners, which are by default 0? Eg. active shaping should add them on the top.

Reply:
Yeah that sounds good. Thanks! We'll merge internally and sync out soon. We'll let you know if we have any other questions.